LINSTORPlugin.pm

```diff
@@ -151,14 +174,53 @@ sub get_preferred_local_node {
     return undef;
 }
 
+sub get_apicrt {
+  my ($cfg) = @_;
+
+  return $cfg->{apicrt} || $default_apicrt;
+}
+
+sub get_apikey {
+  my ($cfg) = @_;
+
+  return $cfg->{apikey} || $default_apikey;
+}
+
+sub get_apica {
+  my ($cfg) = @_;
+
+  return $cfg->{apica} || $default_apica;
+}
+
 sub linstor {
     my ($scfg) = @_;
 
     my @controllers = split( /,/, get_controllers($scfg) );
 
+    my $apicrt = get_apicrt($scfg);
+    my $apikey = get_apikey($scfg);
+    my $apica = get_apica($scfg);
+
+    my %config = ();
+
+    my $proto = "http";
+    my $port = "3370";
+
+    # If cert an key are configured, change protocol and port
+    if ( defined $apicrt and $apikey ) {
```

Comment:
`if ( defined $apicrt and defined $apikey )`?

Reply:
Totally newbie mistake. I'm embarrassed :) Fixed in commit e1c2916.